### DISCUSSION TITLE
Relationships should allow for the same table to be selected

### DISCUSSION THREAD

**Discussion by lylebrown** [Budibase/budibase#1034] [Ideas]
Relationships should allow you to pick the table you are building the relationship in. This will allow for parent/child records in the same table.

**Comment by chrisshaw** [comment]
Any attention being paid to this? Or what is The Budi Way to model a tree? I'm early on but figured I'd get early feedback and leave my thoughts for the next me.

- I have a Groups table.
- In a Group Tree join table, I create Parent and Child 1:N Relationship columns. This way I at least can only select a single record for each Parent and Child.
- The automatically generated columns in the Groups table are weird (in my data model being a Parent to multiple children makes sense, but being a Child to multiple parents doesn't) so I just have to know to ignore them and use the Tree table.

**Reply by joebudi** [reply]
Hey! I am going to create an issue for this and add it to the roadmap. It's a pretty minor change in the Data UI - at the moment we don't let you pick the same table.

**Reply by joebudi** [reply]
You can follow the issue here:
#1373
